truss/templates/trtllm-audio/packages/whisper_trt/modeling.py

```diff
@@ -154,6 +152,23 @@ def generate(
         torch.cuda.synchronize()
 
         decoder_input_ids = decoder_input_ids.type(torch.int32).cuda()
+        if self.decoder_config["plugin_config"]["remove_input_padding"]:
+            # 50256 is the index of <pad> for all whisper models' decoder
+            WHISPER_PAD_TOKEN_ID = 50256
```

Comment:
Where did you find this? I think it should be `50257` which is the endoftext token and is used to pad, but I could be wrong. interested to see the reference.

Reply:
grabbed from [TRT-LLM `run.py`](https://github.com/NVIDIA/TensorRT-LLM/blob/a681853d3803ee5893307e812530b5e7004bb6e1/examples/whisper/run.py#L249-L259). I see https://huggingface.co/docs/transformers/en/model_doc/whisper#transformers.WhisperConfig.pad_token_id as well, what do you think?

Reply:
I think `eos_token` is also the same index 50256

Reply:
I think it would be best to pull out this token from the tokenizer. like here: https://github.com/basetenlabs/fde/blob/main/whisper-trt/src/whisper_trt/model.py#L133
just my two cents